tests/homework_QA_Manager_table.py

```diff
@@ -0,0 +1,71 @@
+import time
+import unittest
+
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
+
+
+class EmployeeList(unittest.TestCase):
+    def setUp(self) -> None:
+        browser = webdriver.Chrome(service=Service(executable_path=ChromeDriverManager().install()))
+        browser.get('http://hrm-online.portnov.com/')
+        self.browser = browser
+    def test_employee_list(self):
+        browser = self.browser
+        browser.find_element(By.CSS_SELECTOR, '#txtUsername').send_keys('admin')
+        browser.find_element(By.CSS_SELECTOR, '#txtPassword').send_keys('password')
+        browser.find_element(By.CSS_SELECTOR, '#btnLogin').click()
+        self.assertIn('/pim/viewEmployeeList', browser.current_url)
+        time.sleep(1)
+        welcome_message = browser.find_element(By.ID, 'welcome').text
+        self.assertEqual('Welcome Admin', welcome_message)
+
+        browser.find_element(By.XPATH, "//select[@id='empsearch_job_title']").click()
+        time.sleep(1)
+        job_title_qa = browser.find_element(By.XPATH, '//option[text()="QA Manager"]').text
+        self.assertEqual('QA Manager', job_title_qa)
+
+        browser.find_element(By.XPATH, '//option[text()="QA Manager"]').click()
+        browser.find_element(By.XPATH, '//input[@id="searchBtn"]').click()
+        time.sleep(2)
+
+        job_title_result = browser.find_elements(By.XPATH, '//td[text()="QA Manager"]')
```

Comment:
This will result in a false positive.
In other words, even if the search fails, and the table still contains all other rows, not just QA Manager(s), your locator will match only QA Managers. So your test will pass even though it should have failed.